cves/CVE-2013-0867.yml

```diff
@@ -90,7 +101,7 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 11c99c78bafa77f679a1a3ba06ad00984b9a4cae
-  note: ''
+  note: 'Fix in the most recent version.'
```

Comment:
Could you elaborate or have a very small summary of the change in plain English?

Reply:
I meant to say the commit was the most recent version published in CVE security.